packages/bff/src/routers/catalogRouter.ts

```diff
@@ -247,12 +247,55 @@ const catalogRouter = (
     )
     .post(
       "/eservices/:eServiceId/riskAnalysis/:riskAnalysisId",
-      async (_req, res) => res.status(501).send()
+      async (req, res) => {
+        const ctx = fromBffAppContext(req.ctx, req.headers);
+        try {
+          await catalogService.updateEServiceRiskAnalysis(
+            unsafeBrandId(req.params.eServiceId),
+            unsafeBrandId(req.params.riskAnalysisId),
+            req.body,
+            ctx
+          );
+          return res.status(204).send();
+        } catch (error) {
+          const errorRes = makeApiProblem(
+            error,
+            bffGetCatalogErrorMapper,
+            ctx.logger
```

Comment:
The operational log is missing

Reply:
✅ done in this [commit](https://github.com/pagopa/interop-be-monorepo/pull/838/commits/3a5fc0336829a46ee2fcd27fa7a8dc17b3d3fcd4)